go/protocol/chat1/extras.go

```diff
@@ -1882,17 +1882,21 @@ func (u UnfurlRaw) String() string {
 	switch typ {
 	case UnfurlType_GENERIC:
 		return u.Generic().String()
+	case UnfurlType_GIPHY:
+		return u.Giphy().String()
 	}
 	return "<unknown>"
 }
 
-func (g UnfurlGenericRaw) String() string {
-	yield := func(s *string) string {
-		if s == nil {
-			return ""
-		}
-		return *s
+func yieldStr(s *string) string {
+	if s == nil {
+		return ""
 	}
+	return *s
+}
+
+func (g UnfurlGenericRaw) String() string {
```

Comment:
maybe we should call this `UnsafeDebugString` so it's only for testing since it would leak info about the websites visited to the log

Reply:
Done!